components/ai_chat/browser/ai_chat_tab_helper.cc

```diff
@@ -141,7 +141,9 @@ void AIChatTabHelper::InitEngine() {
 }
 
 bool AIChatTabHelper::HasUserOptedIn() {
-  return pref_service_->GetBoolean(ai_chat::prefs::kBraveChatHasSeenDisclaimer);
+  base::Time last_seen_disclaimer =
+      pref_service_->GetTime(ai_chat::prefs::kBravekLastSeenDisclaimer);
+  return last_seen_disclaimer.is_null() ? false : true;
```

Comment:
nit:
may be better to use return value, like this:
```
return !last_seen_disclaimer.is_null();
```

Reply:
done